webapp/src/js/util/URL.js

```diff
@@ -0,0 +1,26 @@
+// A class to help create a URL with query parameters.
+class URL {
```

Comment:
Perhaps it is just a pet peeve of mine or there is something I'm missing, but I don't get why this is a class instead of a function that takes an object and returns a string.